.gitignore

```diff
@@ -0,0 +1 @@
+﻿.idea
```

Comment:
I added this because my JetBrains Text Editor (WebStorms) adds a file called .idea and puts all of my text editor preferences in there. Added this into the git ignore so git doesn't commit this into the repository.